### ISSUE TITLE
prevent https from being redirected to http

### ISSUE THREAD

**Issue by raistlinthewiz** [thenativeweb/forcedomain#7]
So I'm using it as following and my site uses the following;

```javascript
app.use(forceDomain({ hostname: 'www.mydomain.com' }));
```

The thing is that it tries to redirect to http version of the same url within https - this should be prevented.

**Comment by goloroden** [comment]
Does this also happen when you additionally specify the `protocol` as shown in the examples in the [Quick Start](https://github.com/thenativeweb/forcedomain#quick-start)?

**Comment by raistlinthewiz** [comment]
Yes it does

**Comment by riversy** [comment]
According to the library code
```
app.use(forceDomain({
  hostname: 'www.mydomain.com',
  protocol: false
}));
```
should help you.

**Comment by goloroden** [comment]
Closed by 418b9fabbd118a9d49c6e3b426ae6ae9dbafdf7e
